src/opentracing-c/span.h

```diff
@@ -0,0 +1,249 @@
+#ifndef OPENTRACINGC_SPAN_H
+#define OPENTRACINGC_SPAN_H
+
+#include <opentracing-c/config.h>
+#include <opentracing-c/destructible.h>
+#include <opentracing-c/value.h>
+
+/** @file */
+
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+
+/**
+ * Span context interface. Span context represents span state that must be
+ * propagated to descendant spans and across boundaries (e.g. trace I:D,
+ * span ID, baggage).
+ * @extends opentracing_destructible
+ */
+typedef struct opentracing_span_context {
+    /** Base class member. */
+    opentracing_destructible base;
+
+    /**
+     * Calls a function for each baggage item in the span context. If the
+     * function returns opentracing_false, it will not be called again, and
+     * foreach_baggage_item will return immediately.
+     * @param span_context Span context instance.
+     * @param f Callback function.
+     * @param arg Argument to pass to callback function.
+     */
+    void (*foreach_baggage_item)(struct opentracing_span_context* span_context,
+                                 opentracing_bool (*f)(void*,
+                                                       const char*,
+                                                       const char*),
+                                 void* arg);
```

Comment:
Could this document what the arguments passed to `f` are?

Reply:
For sure.